recipes/wasmedge/all/conandata.yml

```diff
@@ -1,4 +1,25 @@
 sources:
+  "0.11.1":
+    Windows:
+      "x86_64":
+        Visual Studio:
+          - url: "https://github.com/WasmEdge/WasmEdge/releases/download/0.11.1/WasmEdge-0.11.1-windows.zip"
```

Comment:
Seems a little overkill this is tied to the compiler 🤔 

usually binaries we just delete the compiler so this is odd